libcloud/storage/drivers/azure_blobs.py

```diff
@@ -184,6 +205,8 @@ def _ex_connection_class_kwargs(self):
         # host argument has precedence
         if not self._host_argument_set:
             result['host'] = '%s.%s' % (self.key, AZURE_STORAGE_HOST_SUFFIX)
+        else:
+            result['account_prefix'] = self.key
```

Comment:
consider documenting the keys of the dictionary, as [per the contributing guide](https://libcloud.readthedocs.io/en/latest/development.html#when-returning-a-dictionary-document-its-structure)

Reply:
Done.